packages/react-core/src/components/Form/examples/FormGroupLabelInfo.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Form, FormGroup, TextInput, Popover } from '@patternfly/react-core';
+import HelpIcon from '@patternfly/react-icons/dist/esm/icons/help-icon';
+
+export const FormGroupLabelInfo: React.FunctionComponent = () => {
+  const [value1, setValue1] = React.useState(''); // name
```

Comment:
Looks like you only have one field here, so you could probably just go with `value` or `name`!

```suggestion
  const [name, setName] = React.useState('');
```